systemtest/src/test/java/io/strimzi/systemtest/olm/OlmAllNamespaceST.java

```diff
@@ -21,52 +27,134 @@
 
 @TestMethodOrder(MethodOrderer.OrderAnnotation.class)
 @Tag(OLM)
+@SuiteDoc(
+    description = @Desc("Tests Strimzi deployments managed by OLM when configured to watch all namespaces."),
+    labels = {
+        @Label(TestDocsLabels.OLM)
+    }
+)
 public class OlmAllNamespaceST extends OlmAbstractST {
 
     public static final String NAMESPACE = "olm-namespace";
 
     @Test
     @Order(1)
+    @TestDoc(
+        description = @Desc("Verifies the deployment of a Kafka cluster using the OLM example when the operator watches all namespaces."),
+        steps = {
+            @Step(value = "Deploy Kafka cluster using the OLM example.", expected = "Kafka cluster is deployed and becomes ready."),
+            @Step(value = "Ensure operator (watching all namespaces) manages the Kafka cluster in the designated test namespace.", expected = "Kafka cluster is operational and managed by the operator.")
+        },
+        labels = {
+            @Label(TestDocsLabels.OLM),
+            @Label(TestDocsLabels.KAFKA)
+        }
+    )
     void testDeployExampleKafka() {
         doTestDeployExampleKafka();
     }
 
     @Test
     @Order(2)
+    @TestDoc(
+        description = @Desc("Verifies the deployment of a KafkaUser using the OLM example when the operator watches all namespaces."),
+        steps = {
+            @Step(value = "Deploy KafkaUser using the OLM example.", expected = "KafkaUser is deployed and becomes ready."),
+            @Step(value = "Ensure operator (watching all namespaces) manages the KafkaUser in the designated test namespace.", expected = "KafkaUser is operational and managed by the operator.")
+        },
+        labels = {
+            @Label(TestDocsLabels.OLM)
+        }
+    )
     void testDeployExampleKafkaUser() {
         doTestDeployExampleKafkaUser();
     }
 
     @Test
     @Order(3)
+    @TestDoc(
+        description = @Desc("Verifies the deployment of a KafkaTopic using the OLM example when the operator watches all namespaces."),
+        steps = {
+            @Step(value = "Deploy KafkaTopic using the OLM example.", expected = "KafkaTopic is deployed and becomes ready."),
+            @Step(value = "Ensure operator (watching all namespaces) manages the KafkaTopic in the designated test namespace.", expected = "KafkaTopic is operational and managed by the operator.")
+        },
+        labels = {
+            @Label(TestDocsLabels.OLM),
+            @Label(TestDocsLabels.KAFKA)
```

Comment:
As I mentioned in the first comment, I don't think that `KAFKA` label is needed here.